gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -163,6 +161,89 @@ export default function NxGlobalSidebarPage() {
+                <NxTable.Cell>showSupport</NxTable.Cell>
+                <NxTable.Cell>Boolean</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  If true the support link is shown. At minimum <NxCode>supportLink</NxCode> below should be
+                  provided.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportIcon</NxTable.Cell>
+                <NxTable.Cell>IconDefinition</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  The support link has an icon to its left, by default it is a circle enclosing a question mark
+                  (<NxCode>faQuestionCircle</NxCode>) this specifies an alternate icon to be used.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportText</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  Alternate text that should appear in the support link, by default the text reads "Help and Support".
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportLink</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>URL</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>releaseText</NxTable.Cell>
+                <NxTable.Cell>String</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>Text that appears before the release number.</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>releaseNumber</NxTable.Cell>
+                <NxTable.Cell>String</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>A release number.</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>productText</NxTable.Cell>
+                <NxTable.Cell>String</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>Text string to denote product family or other custom text.</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>showSonatype</NxTable.Cell>
+                <NxTable.Cell>Boolean</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  If true the "Created by Sonatype" tagline is displayed unless alternate text is provided below.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>sonatypeText</NxTable.Cell>
```

Comment:
What's the use case for this customization?  If arbitrary additional content is needed we should probably define a place for that elsewhere

Reply:
> What's the use case for this customization? If arbitrary additional content is needed we should probably define a place for that elsewhere

I was just covering my bases, but we could make the wording of this one mandatory.

Reply:
> but we could make the wording of this one mandatory.

That would make sense to me

Reply:
> > but we could make the wording of this one mandatory.
> 
> That would make sense to me

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/9e9973513a85f2a33fbe3404ba77ebb2c5b3481c